documentation/project-docs/SDK-PR-guide.md

```diff
@@ -26,7 +26,7 @@ After the minor release locks down, it transitions into servicing / "QB" (Quarte
 ### Servicing releases
 The .NET SDK has monthly servicing releases (e.g.`7.0.100` VS `7.0.101`) aligning with the .NET Runtime servicing releases. These are for top fixes and security updates only to limit risk.
 Any servicing release is open for check-ins from the day the [branding PRs](https://github.com/dotnet/sdk/pulls?q=is%3Apr+branding) are merged (~1st of each month) and when code complete is (typically two weeks later).
-The servicing branches are locked from the time of code complete to the next branding in case we need to respin any monthly release. Final signoffs are typically in the last week of each month.
+The servicing branches are locked from the time of code complete to the next branding in case we need to respin any monthly release. We use the `Branch Lockdown` label to mark PRs that should go into the next servicing release but the branch is currently not open. Final signoffs are typically in the last week of each month.
```

Comment:
Is there a way for us to have some automated action can calculate lockdown dates and have the labels automatically apply when a PR is created? The dates usually vary based on when the release date is, etc. so I'm guessing it's not easy, but perhaps it can be scoped to really critical periods

Reply:
That would be pretty awesome.

Reply:
I added a workflow yml that should add the label to PRs by maestro to the release/8 and release/9 branches (excluding release/9.0.3xx) as long as it's after the third Tuesday of the month or before the first Tuesday of the month (when branding happens). I don't know how to test this though.